cns/service/main.go

```diff
@@ -475,6 +506,10 @@ func main() {
 		logger.InitAI(aiConfig, ts.DisableTrace, ts.DisableMetric, ts.DisableEvent)
 	}
 
+	if telemetryDaemonEnabled {
+		go startTelemetryService(rootCtx)
```

Comment:
since we are not returning any error, failure to start telemetry service will go undetected. Should we exit the CNS if we can't start telemetry service? It will provide early failure. Otherwise we may not have telemetry data to debug an issue w/o telemetry

Reply:
hmm thats more aggresive imo. also i dont want to change existing behaviour. this telemetry service is intended only for cni and cns telemetry not dependent on this.. if we couldn't get cni logs in worst case, we can ask cx to fetch logs.. even if telemetry service is not started, cns is functional, so i dont think exit would be good option here